evap/evaluation/models.py

```diff
@@ -559,6 +559,13 @@ def general_contribution_has_questionnaires(self):
 
     @property
     def all_contributions_have_questionnaires(self):
+        if is_prefetched(self, "contributions"):
+            if not self.contributions:
+                return False
```

Comment:
Not `True`? The line below is `not contributions....exists()`, so if there are no contributions, we should default to `True`, right?

We could also delegate the universal qualifier to the `all` below by also taking this branch if `not self.contributions`

Reply:
Yeah this is weirdness regarding the general contribution and how a "general contribution does not exist" case should be handled when asking for "do all contributions have questionnaires?".

We have a test that asserts that we `return False` here. If the general contribution does not yet exist, it can't have questionnaires assigned, so callers of this method probably expect `False` as a return value that they likely interpret as "can we proceed with this evaluation?"

This method currently fails this test. I'd wait with further changes here until we have a conclusion on how to handle general contributions in #1839.

Reply:
Hm, I think if we want to move forward with this PR, we should just stick to the odd `if not self.general_contribution` for now (and leave a note in the other issue)